app/components/Sidebar/components/Version.tsx

```diff
@@ -2,41 +2,36 @@ import * as React from "react";
 import { useTranslation } from "react-i18next";
 import styled from "styled-components";
 import Badge from "~/components/Badge";
-import { version } from "../../../../package.json";
+import Logger from "~/utils/Logger";
+import { version as currentVersion } from "../../../../package.json";
 import SidebarLink from "./SidebarLink";
 
 export default function Version() {
-  const [releasesBehind, setReleasesBehind] = React.useState(-1);
+  const [versionsBehind, setVersionsBehind] = React.useState(-1);
   const { t } = useTranslation();
 
   React.useEffect(() => {
-    async function loadReleases() {
-      const res = await fetch(
-        "https://api.github.com/repos/outline/outline/releases"
-      );
-      const releases = await res.json();
+    async function loadVersionInfo() {
+      try {
+        // Fetch version info from the server-side proxy
+        const res = await fetch("/api/installation.info", {
```

Comment:
Oh, I just realised this should use `ApiClient` (see other usage) so you don't need to define headers, parse JSON, etc.

Reply:
Done, check it [here](https://github.com/outline/outline/pull/7744/commits/de0a99e090996fe37d77efb06029d27b3504f6c1)